code/modules/organs/organ.dm

```diff
@@ -201,7 +201,7 @@
 
 /obj/item/organ/proc/handle_ailment(var/datum/ailment/ailment)
 	if(ailment.treated_by_reagent_type)
-		for(var/datum/reagents/source in list(owner.get_injected_reagents(), owner.reagents, owner.get_ingested_reagents()))
+		for(var/datum/reagents/source in list(owner.get_injected_reagents(), owner.reagents, owner.get_ingested_reagents(), owner.get_inhaled_reagents()))
```

Comment:
This will be iterating the same list several times for nonhumans if they ever get organs. Not really blocking, as above.